views/users/form.blade.php

```diff
@@ -12,6 +12,7 @@
 
     <x-twill::input
         name="email"
+        :placeholder="twillTrans('twill::lang.user-management.email')"
```

Comment:
Thanks @tiagofrancafernandes! I'm not sure I understand why a placeholder is needed here.